src/main/java/org/skriptlang/skript/bukkit/recipes/elements/ExprRecipeKeyConverter.java

```diff
@@ -0,0 +1,73 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.NamespacedUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+
+@Name("Recipe Key Converter")
+@Description({
+	"Converts a string to a valid key to be used with recipes.",
+	"'using minecraft namespace' will convert the string to a minecraft namespace key"
+})
+@Examples({
+	"send the converted recipe key from \"my_recipe\"",
+	"loop all custom recipes:",
+		"\tif recipe key of loop-recipe = converted key from \"my_recipe\" using minecraft namespace:",
+			"\t\tbroadcast loop-recipe",
+	"",
+	"register a new shaped recipe with the key (converted key from \"my_recipe\"):",
+		"set the recipe ingredients to diamond, iron ingot and raw copper",
+		"set the recipe result to diamond helmet"
+})
+@Since("INSERT VERSION")
+public class ExprRecipeKeyConverter extends SimpleExpression<String> {
```

Comment:
Can you explain to me the need for this?

Reply:
I made that because attempting to do `if recipe key of %% = "my_recipe":` would never pass as the recipe key would always contain a namespace i.e. `skript:` , `minecraft:`
Fuse told me to ask about a comparator for that but I had made this in tandom with the tests for the `NamespacedUtils#createNamespacedKey` as per Pikachu's code. So I assumed it would be "useful" to a degree.

Reply:
I don't think it's any more useful than just typing `"minecraft:my_recipe"`